react/Breadcrumb.tsx

```diff
@@ -41,11 +41,11 @@ class Breadcrumb extends Component<Props> {
     const categoriesSorted = categories
       .slice()
       .sort((a, b) => a.length - b.length)
-    return categoriesSorted.map(category => {
-      const categoryStripped = category.replace(/^\//, '').replace(/\/$/, '')
+    return categoriesSorted.map((category) => {
+      let categoryStripped = category.replace(/^\//, '').replace(/\/$/, '')
       const categories = categoryStripped.split('/')
       const [categoryKey] = categories.reverse()
-
+      categoryStripped = categories.length === 1 ? `${categoryStripped}/d` : categoryStripped
```

Comment:
You could use the `+=` operator, but that's fine

e.g: `categoryStripped += categories.length === 1 ? '/d' : ''`

Reply:
Nice, thanks!